app/src/main/res/drawable-anydpi/ic_light_bulb.xml

```diff
@@ -0,0 +1,11 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Let's remove the light-bulb icon (all of them) and just use the app icon instead.